docs/pages/play.vue

```diff
@@ -15,149 +14,222 @@
         </p>
       </div>
       <div class="col-12">
-        <div v-if="loading" class="alert alert-info show text-center">
+        <div
+          v-if="loading"
+          class="alert alert-info show text-center"
+        >
           <strong>Loading JavaScript Compiler...</strong>
         </div>
-        <div v-else class="clearfix">
-          <form class="d-inline-block ml-2 mr-0 p-0 float-right"
-                method="post"
-                action="https://jsfiddle.net/api/post/library/pure/"
-                target="_blank">
-            <input type="hidden" name="html" :value="fiddle_html">
-            <input type="hidden" name="js" :value="fiddle_js">
-            <input type="hidden" name="resources" :value="fiddle_dependencies">
-            <input type="hidden" name="css" value="body { padding: 1rem; }">
-            <input type="hidden" name="js_wrap" value="l">
-            <b-btn size="sm" type="submit" :disabled="!isOk">Export to JSFiddle</b-btn>
+        <div
+          v-else
+          class="clearfix"
+        >
+          <form
+            class="d-inline-block ml-2 mr-0 p-0 float-right"
+            method="post"
+            action="https://jsfiddle.net/api/post/library/pure/"
+            target="_blank"
+          >
+            <input
+              type="hidden"
+              name="html"
+              :value="fiddle_html"
```

Comment:
I still prefer that things like this could stay on a single line (if htey fint on a single line).  As now six lines on screen have changed to 30 lines on screen.  Making the user (i,e. me :stuck_out_tongue: ) have to scroll up/down a lot more (when viewing or editing the file) in order to see what that component is doing.

Is there ea rule that can say if it fits in 80-100 characters, leave it alone?

Reply:
I currently don't know why increasing `vue/max-attributes-per-line` to 3 doesn't help here :/

Reply:
Is there a maximum line width rule coming into play here?

Reply:
I don't thin so. They are < 100 chars.

Reply:
Weird...